integration-tests/custom-networks.test.ts

```diff
@@ -0,0 +1,55 @@
+import {createWallet, IWallet} from '@docknetwork/wallet-sdk-core/src/wallet';
+
+describe('Custom networks', () => {
+  let wallet: IWallet;
+  const networks = [
+    {
+      name: 'polygon',
+      id: 'polygon',
+      configs: '{}',
+    },
+    {
+      name: 'Polygon mumbai',
+      id: 'mumbai',
+      configs: '{}',
+    },
+  ];
+
+  it('should create a wallet with custom networks', async () => {
+    wallet = await createWallet({
```

Comment:
is there anything we could assert about the wallet to make sure it was created properly?

Reply:
Yes, I've updated the unit test. The wallet will be created in a beforeEach function. An empty wallet for each test